test/cppcheck_suppressions.txt

```diff
@@ -21,7 +21,12 @@ shiftNegative:src/utils/msc_tree.cc
 
 // 
 // ModSecurity v3 code...
-// 
+//
+
+functionConst:src/utils/shared_files.h:60
+useInitializationList:src/utils/shared_files.h:88
```

Comment:
Similar here, but it seems that file only has [77](https://github.com/owasp-modsecurity/ModSecurity/blob/b0a56941fab7bc50bd13203dc11810231392e797/src/utils/shared_files.h#L77) lines, so adding line 88 makes no sense.